docs/writing-rules/experiments/extract-mode.md

```diff
@@ -91,8 +91,9 @@ Extract mode rules **also require** two additional fields:
   - `extract`
   - `dest-language`
 
-Extract mode has one **optional** field:
+Extract mode has two **optional** field:
   - `reduce`
+  - `json`
```

Comment:
here it's actually transform:
Thinking about it we should have called it  map, so we have a mapreduce inside extract mode